src/components/ExtraPages/Help/PlotHelp.js

```diff
@@ -1,39 +1,365 @@
+                  Points close to the x-axis represent runs with high
+                  correlation coefficients, indicating that those runs produce
+                  similar spatial patterns to those in the reference
+                  climatology; points that lie closer to the $y$-axis represent
+                  runs with correlation coefficients close to zero, indicating
+                  very different spatial patterns to those in the reference
+                  field.
+                </p>
+                <p>
+                  Runs represented by points closer to the black star more
+                  closely replicate the the reference climatology in terms of
+                  both the location of spatial features and the range of values
+                  simulated across the UK. During the evaluation period, this is
+                  a desirable trait, indicating that the runs produce realistic
+                  spatial climatologies. For future periods, this indicates that
+                  the spatial patterns are very similar to those produced during
+                  the baseline period. The radial axes are scaled in such a way
+                  that the distance from each point to the black star is
+                  proportional to the centred root mean squared error in the
+                  simulated field; however, note that this centred root mean
+                  squared error excludes UK-wide biases in the runs, which must
+                  be assessed separately using boxplots.
+                </p>
+              </Row>
+            </Row>
+          </div>
+          <br></br>
+          {/* anova */}
+          <div>
+            <h5 id="anova">
+              Maps of sources of variation in the EuroCORDEX ensemble
+            </h5>
+            <p>
+              Analysis of variance (ANOVA) can be used to partition the
+              variation within the EuroCORDEX ensemble into contributions that
+              can be attributed to either the GCMs or RCMs. The ANOVA plots
+              consist of a map of the standard deviation of the model
+              climatologies (or changes in the climatologies) within the
+              EuroCORDEX ensemble, with darker areas indicating regions of
+              higher variability, where there is less agreement between the
+              ensemble members; and of maps of the proportion of this
+              variability attributed to systematic differences between the GCMs
+              and the RCMS, and of the residual variability. The plot titles
+              also give the proportion of the variance over the whole of the UK
+              explained by each component: note that, due to the unbalanced
+              structure of the EuroCORDEX ensemble (as seen in the table of
+              available GCM-RCM pairs at{" "}
+              <Link to="link-to-ensembles-page">link-to-ensembles-page</Link>,
```

Comment:
```suggestion
              structure of the EuroCORDEX ensemble (as seen in the{" "}
              <Link to="data-help#eurocordex">table of
              available GCM-RCM pairs</Link>,
```

Reply:
this needs to be a `HashLink` so I'll make this change locally